recipes/openmm-ml/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "openmm-ml" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmm/openmm-ml/archive/28b4fadd761221fbe085570faac3e95a2d0be4e0.tar.gz
+  sha256: 913c48641c5fdcbdc6d7ccf14cd33a20ad432c781455e331da86a7a8cc455330
```

Comment:
I think it's cleaner to use `git_url` and `git_rev`.  Here is [an example](https://github.com/conda-forge/openmm-feedstock/blob/18d41142a757522eb291614d937c46a67be084c7/recipe/meta.yaml#L12-L13).  `git_rev` can be either the tag for a release or the hash for revision.

Reply:
I agree, and this is something that really isn't clear in the docs:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos
So I default to using a tarball, but clearly there are examples (like the one you linked) so I will change it and see what they say when they review it, thanks!